platform/android/export/gradle_export_util.h

```diff
@@ -142,4 +142,28 @@ Error _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset
 	return OK;
 }
 
+String bool_to_string(bool v) {
+	return v ? "true" : "false";
+}
+
+String _get_gles_tag() {
+	bool min_gles3 = ProjectSettings::get_singleton()->get("rendering/quality/driver/driver_name") == "GLES3" &&
+					 !ProjectSettings::get_singleton()->get("rendering/quality/driver/fallback_to_gles2");
+	return min_gles3 ? "    <uses-feature android:glEsVersion=\"0x00030000\" android:required=\"true\" />\n" : "";
```

Comment:
Seems it's missing an option for `gles2`.

Reply:
gles 2 is included in the base android manifest file, so by not including anything on the variant manifest file, it defaults to gles2.